8510Final-AR.qmd

```diff
@@ -0,0 +1,311 @@
+---
+title: "Final Project"
+author: "Megha Khanna"
+date: "05/01/2024"
+---
+# Title of paper here
+
+For my Final project, I have applied codes to analyze a dataset of newspaper publications concerning Jack the Ripper and his notorious murders.
+
+My research questions are as follows:
+
+1.  How does the frequency and context of terms associated with Jack the Ripper differ from those related to his victims in historical newspaper articles, and what does this tell us about societal attitudes towards crime and victimhood in late Victorian England?
+
+2.  How did the topics covered in newspaper reports change before and after 1888, the year of Jack the Ripper's crimes? What can this reveal about the influence of these events on public discourse and media focus?
+
+## Introduction
+
+The infamous Jack the Ripper murders of 1888, targeting several sex workers in London's Whitechapel district, became a stark illustration of the conversation between media sensationalism, public fear, and the stigmatization of sex workers. Jack the Ripper's name continues to resonate throughout criminal history, not just for the horrific nature of the crimes but also for the media frenzy that erupted in late Victorian England.
+
+This analysis aims to explore the differential portrayal of Jack the Ripper compared to his victims through the lens of historical newspaper articles. By analyzing the frequency and context of words used, we can gain insights into the narrative and societal perceptions constructed by the media of that era. The other half of the analysis looks at the pre and post year 1888 and how the narratives related to crimes have significantly changes with the sensationalism of ripper crimes.
+
+## Historical Background
+
+The analysis delves into the socio-political atmosphere of London in the late 19th century, emphasizing the role of media in shaping the narrative around the Ripper murders. Walkowitz points out that the press sensationalized the events, thereby embedding a fear of male violence deep within the public consciousness. This sensationalism contributed to a moral panic that framed the city as a perilous place for women​​. Walkowitz argues that over the past century, Jack the Ripper has been transformed into a mythical figure representing male violence against women. This transformation has permeated cultural narratives, impacting how society views and treats women, especially in urban settings​​.(Walkowitz, 544)
+
+## Data Manipulation and analyzing the Word Frequencies
+```{r}
+#| echo: false
```

Comment:
I want you to go take a look at other articles that use digital methods and think about the way that they are balancing historical context and code. See my email for more on this.

Do the libraries need to be here for you to make an argument? Probably not. But they are needed in order for your code to run. One way to fix this is to use the ```#| echo: false``` option to have this run in the background but not actually display in your final paper.